src/pages/Appointments/components/Calendar.jsx

```diff
@@ -0,0 +1,19 @@
+import React, { } from 'react';
+import Calendar from 'react-calendar';
+import 'react-calendar/dist/Calendar.css';
+import './calendarStyles/calendar.css';
+
+export function StyledCalendar({ value, onChange }) {
+  return (
+    <div className="calendar-container">
+      <Calendar
+        value={value}
+        onClickDay={onChange}
+        locale="en-US"
+        minDate={new Date()}
+        formatShortWeekday={(locale, date) => ['S', 'M', 'T', 'W', 'T', 'F', 'S'][date.getDay()]}
```

Comment:
declare handler upper and reuse here 
['S', 'M', 'T', 'W', 'T', 'F', 'S'] should be inside constants

Reply:
also wrap with useCallback hook